browser/search_engines/search_engine_tracker.cc

```diff
@@ -106,5 +155,27 @@ void SearchEngineTracker::OnTemplateURLServiceChanged() {
     if (url != default_search_url_) {
       RecordSearchEngineP3A(url, template_url->GetEngineType(search_terms));
     }
+    RecordSwitchP3A(url);
+  }
+}
+
+void SearchEngineTracker::RecordSwitchP3A(const GURL& url) {
+  // Default to the last recorded switch so when we're called
+  // at start-up we initialize the histogram with whatever we
+  // remember from the previous run.
```

Comment:
Is this intentional? Currently if a user changes the search and restarts the browser for a week, we will send a histogram `kSomethingToSomething` on each browser launch. Doesn't it break the stats we're trying to carefully aggregate?